curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6477062778c85972eb648030.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 6477062778c85972eb648030
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+You should now center the cat head. 
+
+Give the `cat-head` element a position property set to `absolute`. Then set a value of `0` for the `right`, `left`, `top`, `bottom` properties, then set its `margin` property on all sides to `auto`. That's one way to center an element vertically and horizontally using CSS positioning.
+
+You should also remove the `.box` CSS rule and its declarations because you don't need them anymore.
```

Comment:
If we are asking them to remove the box rule, it should be in the editable region.